### PR TITLE
Bridge 537

### PR DESCRIPTION
**Pull request by EricSiegNW**
Store height/weight in keychain rather than HK, due to issue when write permission is denied

### CODE REVIEW

File: APCAppCore/APCAppCore/DataSubstrate/Model/APCUser.m [hunk 1]
```diff
@@ -359,6 +361,75 @@ - (void)setSubpopulationGuid:(NSString *)subpopulationGuid {
...
+        HKQuantity *height = [HKQuantity quantityWithUnit:[HKUnit meterUnit] doubleValue:heightDouble];
+        return height;
+    }
+}
+
+- (void)setHeight:(HKQuantity *)height
+{
+    double heightDouble = [height doubleValueForUnit:[HKUnit meterUnit]];
+    NSString *heightString = [NSString stringWithFormat:@"%lf", heightDouble];
+    [APCKeychainStore setString:heightString forKey:kHeightPropertyName];
```
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/DataSubstrate/Model/APCUser.m:395]: Does `APCKeychainStore` not support saving an `NSNumber` or `NSSecureCoding`?

Comment (EricSiegNW) [reply APCAppCore/APCAppCore/DataSubstrate/Model/APCUser.m:395]: Nope, just strings currently.

Comment (syoung-smallwisdom) [reply APCAppCore/APCAppCore/DataSubstrate/Model/APCUser.m:395]: weird.

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

**Pull request merged**